packages/webdriver-utils/src/providers/genericProvider.js

```diff
@@ -120,9 +80,7 @@ export default class GenericProvider {
 
     this.addDefaultOptions();
 
-    const percyCSS = (this.defaultPercyCSS() + (this.options.percyCSS || '')).split('\n').join('');
-    log.debug(`[${name}] : Applying the percyCSS - ${this.options.percyCSS}`);
-    await this.addPercyCSS(percyCSS);
+    this.options.percyCSS = (this.options.percyCSS || '').split('\n').join('');
```

Comment:
query: why are we replacing new line with nothing?

Reply:
Yes its kind of serializing, I tested once earlier it was not working, so i kept this flow as it is. I guess if we are not adding anything it should run fine. When we are adding things then it needs such thing.

Reply:
weird 😕  anyways..